nrf_security/CMakeLists.txt

```diff
@@ -49,6 +47,10 @@ else()
   message(FATAL_ERROR "west not installed, please provide ARM_MBEDTLS_PATH to CMake to support security libraries")
 endif()
 
+set(mbedcrypto_glue_include_path
+  "${CMAKE_CURRENT_LIST_DIR}/include/mbedcrypto_glue/mbedtls"
```

Comment:
Use absolute paths based on nrf dir or zephyr dir.

Reply:
Okayed by @tejlmand (CMAKE_CURRENT_LIST_DIR children is still "absolute")